<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="dropBox_algorithmSelector">
      <item>
       <property name="text">
        <string>Bubble Sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Insertion Sort</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_togglePlay">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_step">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBubbleSort">
   <property name="text">
    <string>Bubble Sort</string>
   </property>
  </action>
  <action name="actionInsertionSort">
   <property name="text">
    <string>Insertion Sort</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
